doc/classes/NativeMenu.xml

```diff
@@ -720,7 +728,76 @@
 				[b]Note:[/b] This method is implemented only on macOS.
 			</description>
 		</method>
+		<method name="set_system_menu_hidden">
+			<return type="void" />
+			<param index="0" name="menu_id" type="int" enum="NativeMenu.SystemMenus" />
+			<param index="1" name="hidden" type="bool" />
+			<description>
+				Hides/shows the system menu. Only system menus with no default items can be hidden. When it is hidden, an item does not appear in a menu and its action cannot be invoked.
```

Comment:
```suggestion
				Hides/shows the system menu with the given [param menu_id]. Only system menus with no default items can be hidden. When it is hidden, an item does not appear in a menu and its action cannot be invoked.
```

_"When it is hidden, an item does not appear in a menu and its action cannot be invoked."_ is rather ambiguous. "It is hidden" probably refers to "an item", but all this time we've been talking about "system menu", so what's the subject here?

Reply:
Copy-paste from `set_item_hidden`, this part is probably redundant here.